modules/database/src/ioc/db/menu.h

```diff
@@ -0,0 +1,52 @@
+/*************************************************************************\
+* SPDX-License-Identifier: EPICS
+* EPICS BASE is distributed subject to a Software License Agreement found
+* in file LICENSE that is included with this distribution.
+ \*************************************************************************/
+
+/** @file menu.h
+ * @brief Adds db menu capabilities
+ * @see @ref menu
+ */
+
+#ifndef INCmenuH
+#define INCmenuH
+
+#include "menuFtype.h"
+#include "epicsTypes.h"
+
+/*Returns the variable size of the menuFtype*/
+epicsEnum16 menuEpicsTypeSizes(menuFtype type)
```

Comment:
For future reference... 

Qualifying with `static inline` would be necessary to avoid duplicate symbol errors during linking.  Something which would likely become apparent were this header included/used from one or more object files.  (eg. like some unit test code, hint hint)

While unnecessary with `static inline`, the usual `#ifdef __cplusplus` ... `extern "C" {` would not be out of place in this header.  For that matter, a new header doesn't seem necessary to me.  This function could be added in `menuFtype.dbd.pod`.  See eg. `struct aidset` defined in `aiRecord.dbd.pod`.

Also, I think that `size_t` should be returned.  This is the type of `sizeof()`, and I doubt a narrowing cast would have any practical benefit.